src/Nethermind/Nethermind.Core/Specs/IReleaseSpec.cs

```diff
@@ -360,6 +359,22 @@ public interface IReleaseSpec : IEip1559Spec, IReceiptSpec
 
         public bool IsBeaconBlockRootAvailable => IsEip4788Enabled;
         public bool MCopyIncluded => IsEip5656Enabled;
+
+        /// <summary>
+        /// AuRaSystemCalls - true
```

Comment:
How rules described below is different from just having `EIP158 disabled` applied to system calls?